Sources/Bar.swift

```diff
@@ -0,0 +1,30 @@
+
+import Foundation
+
+/// Serves anything you order.
+/// `Bar` tries to map any JSON to a `Mappable` type.
+public class Bar {
+    public let service: Service
+
+    public init (service: Service) {
+        self.service = service
+    }
+
+    /// Receives JSON from the a `Service` and maps this to a `Result` of type `M` that is Mappable.
+    /// Typ! You can subclass 'Bar' and add a default service
+    /// - parameter order : gives the details to find the entity on the server
+    /// - parameter service : default = `JSONService` fires the `Order` to a server
+    /// - parameter result : enum containing the requested entity of type `M` on succes or a failure.
+    public func serve <M: Mappable> (order: Order, result: (Result <M>) -> ()) {
```

Comment:
❓Is mappable already needed here? What if I just want to read the JSON value and no model is available? Or are we to make models explicitly required when using `Faro`? Which is good I think, it's like this in Bolero, and I kinda like it.